### PR TITLE
[FIF-372] Updated Privacy page to load privacy description from md file

### PR DESCRIPTION
**Pull request by jleiva-gap**
- The content of the privacy page was moved to an external Markdown file.
- To modify the Privacy Policy and test the change, you must update the content of the **PRIVACY.POLICY.md** file and go to http://<site>/app.privacypolicy.

### CODE REVIEW

File: EdFi.Buzz.UI/src/Feature/Privacy/index.tsx [hunk 1]
```diff
@@ -32,137 +34,50 @@ const BackToHome = styled(Link)`
...
-      <div>Go back to Home</div>
-    </BackToHome>
-  </MainContainer>
-);
+const Privacy: React.FunctionComponent<PrivacyProps> = ({title}) => {
+  const [markdownFile,setMarkdownFile] = useState(null);
+
+  useEffect(() => {
+    let isMounted = true; // note this flag denote mount status
+    const readmePath = (`http://${window.location.host}/PRIVACY.POLICY.md`);
```
Comment (kurtjohnson-HS) [inline EdFi.Buzz.UI/src/Feature/Privacy/index.tsx:42]: http protocol will cause issues if the user does not enable origin that with CORS. Can we omit that?

**Review by kurtjohnson-HS — CHANGES_REQUESTED** [review]

**Comment by kurtjohnson-HS** [comment]
There also needs to be a mimeMap for the .md file extension in the web.config to serve that file on IIS.

File: EdFi.Buzz.UI/src/Feature/Privacy/index.tsx (hunk 1 above)
Comment (jleiva-gap) [reply EdFi.Buzz.UI/src/Feature/Privacy/index.tsx]: Updated to use window.location.origin instead of the hardcoded http.

**Comment by jleiva-gap** [comment]
> 
> 
> There also needs to be a mimeMap for the .md file extension in the web.config to serve that file on IIS.

Added mimeMap for the .md file.

**Review by kurtjohnson-HS — APPROVED** [review]

**Pull request merged**